pkgs/tools/misc/dateutils/default.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  version = "0.4.1";
+	name = "dateutils-${version}";
+
+  src =fetchurl {
```

Comment:
Is this the preference? It can't find the make file correctly when I use the `fetchFromBitbucket`.